### DISCUSSION TITLE
Adjust sidebar/toc width

### DISCUSSION THREAD

**Discussion by elsampsa** [pradyunsg/furo#507] [Ideas] [answered]
I have lots of nesting in the left sidebar & would like to make it wider.  Tried to experiment to no avail:
```
html_theme_options = {
    # 'page_width' : '60%', # not in furo template 
    'sidebarwidth': '80%', # sphinx default option, but no effect whatsoever in here
    'body_max_width': 'auto',
    "sidebar_hide_name": True,
    "light_css_variables": {
        "color-brand-primary": "#00aaee",
        "color-brand-content": "#00aaee",
    },
}
```
Any idea how to achieve wider sidebar?  :) 

And thanks for the great template!

**Comment by pradyunsg** [comment, marked as answer]
This isn't possible with Furo.

The sidebar width (as well as other aspects of the layout) are hard-coded to ensure that it's consistent and to reduce (a) the surface area of complexity in the design and (b) to avoid the possibility of downstream documentation authors tweaking things in a way that degrades the user experience.

This is one of the opinionated design choices made as a part of the theme's design.
